easybuild/easyconfigs/g/Gctf/Gctf-1.06.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'Gctf'
+version = '1.06'
+
+homepage = 'https://www.mrc-lmb.cam.ac.uk/kzhang/'
+description = """Gctf: real-time CTF determination and correction, Kai Zhang, 2016"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://www.mrc-lmb.cam.ac.uk/kzhang/Gctf/']
+sources = [
+    '%(name)s_v%(version)s_and_examples.tar.gz',
+]
+patches = [('Gctf-wrapper.sh', '.')]
+checksums = [
+    'bfb6305f36571f895402bb22a2e8c6c5c4f13e28d3354ff8fe101ec25b351d77',  # Gctf_v1.06_and_examples.tar.gz
+    'bd019bd752ac56b89f9f09ce8e9aab41977be1078145b7afa5faf48f4ef4781b',  # Gctf-wrapper.sh
```

Comment:
This is failing for me:
```
== 2019-05-14 08:46:10,294 filetools.py:724 DEBUG Using blocksize 16777216 for calculating the checksum
== 2019-05-14 08:46:10,294 filetools.py:772 DEBUG Computed sha256 checksum for /tmp/eb-p8hqwK/files_pr8045/g/Gctf/Gctf-wrapper.sh: 62cc5b011476a5005360003c16bcc408b09f18e368d261b95522715d6bc47617 (correct checksum: bd019bd752ac56b89f9f09ce8e9aab41977be1078145b7afa5faf48f4ef4781b)
== 2019-05-14 08:46:10,320 build_log.py:163 ERROR EasyBuild crashed with an error (at ?:124 in __init__): Checksum verification for /tmp/eb-p8hqwK/files_pr8045/g/Gctf/Gctf-wrapper.sh using bd019bd752ac56b89f9f09ce8e9aab41977be1078145b7afa5faf48f4ef4781b failed. (at easybuild/framework/easyblock.py:1777 in checksum_step)
== 2019-05-14 08:46:10,320 easyblock.py:3010 WARNING build failed (first 300 chars): Checksum verification for /tmp/eb-p8hqwK/files_pr8045/g/Gctf/Gctf-wrapper.sh using bd019bd752ac56b89f9f09ce8e9aab41977be1078145b7afa5faf48f4ef4781b failed.
== 2019-05-14 08:46:10,320 easyblock.py:293 INFO Closing log for application name Gctf version 1.06
```

Reply:
Crap, forgot to update checksum after the change in the comment...

Reply:
Done